### DISCUSSION TITLE
The question of competition time schedule

### DISCUSSION THREAD

**Discussion by gaozhanfire** [pykt-team/pykt-toolkit#88] [Q&A] [answered]
When will the private leaderboard be visible?I noticed that the private leaderboard was not open after 11: 59 UTC.

**Comment by meta-tabchen** [comment, marked as answer]
We still working on aggregate team info, after merging we will open the private leaderboard.

**Reply by gaozhanfire** [reply]
Dear pykt team
First of all, I wish you a happy new year! Thank you for your efforts during the holiday. But, what's the progress of collecting information about all teams? This information is very important to our team, because our team is stuck on the winning line, and many team members are still anxiously waiting for the result. Can you help me answer this question? Thank you again for your efforts.

**Reply by meta-tabchen** [reply]
We have released private leaderboards so you can check your team's results now.

**Reply by gaozhanfire** [reply]
Thank you for not releasing private leaderboards during the holiday.😭😂😭😂
